sdk/python/feast/infra/feature_servers/multicloud/Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM python:3.8
 RUN apt update && \
         apt install -y jq
 RUN pip install pip --upgrade
-RUN pip install "feast[aws,gcp,snowflake,redis,go]"
+COPY . .
+
+RUN pip install -r requirements.txt
```

Comment:
I'd want to make sure that any native libraries that the mysql client needs are also installed here (e.g. the stuff listed here: https://github.com/PyMySQL/mysqlclient#linux)

Reply:
Added native libraries in [this commit](https://github.com/feast-dev/feast/pull/3223/commits/6b283a694270443f5141f36c4312ea885b0193d7)